app/controllers/session_controller.rb

```diff
@@ -0,0 +1,22 @@
+class SessionsController < ApplicationController
+
+    def create
```

Comment:
i don't think the create method is triggered in your code flow. u can delete this method